chrome/ScreenSharing/README.md

```diff
@@ -88,7 +88,7 @@ Register you extension:
 
 ```javascript
 
-OT.registerScreenSharingExtension('chrome', 'YOUR-EXTENSION-ID');
+OT.registerScreenSharingExtension('chrome', 'YOUR-EXTENSION-ID', 2);
```

Comment:
Also, you need to modify this line (adding the new parameter):

https://github.com/opentok/screensharing-extensions/blob/master/screensharing-test.html#L46